test-data/unit/stubgen.test

```diff
@@ -2963,6 +2999,8 @@ __version__ = ''
 [out]
 from m import __version__ as __version__
```

Comment:
This was a problem before your PR, but I'm not sure what the justification is for not importing `__about__` and `__author__` here, given that they're included in `__all__` at runtime

Reply:
Yes, they're in IGNORED_DUNDERS in stubutil.py. It generally makes sense to exclude these names as they're not useful in stubs, but we shouldn't ignore them if they're in `__all__`.

Reply:
Pushed a change so that IGNORED_DUNDERS aren't ignored if they're in `__all__`.